gym/envs/phys2d/bipedal_walker.py

```diff
@@ -0,0 +1,731 @@
+from typing import Optional
+
+import brax
+import jax
+import numpy as np
+from brax import jumpy as jp
```

Comment:
Our plan is that jumpy is solely used by the wrappers for easy compatibility between numpy and jax. 
But with the environment, to ensure performance, we should only use jax.numpy for operations.

We will provide jax based spaces